src/node/isometric_map.cpp

```diff
@@ -80,10 +80,15 @@ void IsometricMap::_notification(int notif) {
 }
 
 void IsometricMap::_on_enter_tree() {
+    static bool initialized{false};
```

Comment:
I'm confused, wouldn't that only work for the very first map you are loading in the game but then all the next ones wouldn't have any child added ?

Reply:
Not in constructor, it wouldn't work with loading a map from a scene file as the object would be created first using the constructor and then the property containing the proper data set at a later point. 
We can do that in set_grid_3d, this function is only there for file reading purpose anyway as it's not the property is attached to is not available in the exposed API of the object.

Reply:
Using `set_grid_3d` for this purpose makes it crash as space is not here when this methos is called.
I added a private field to make the check